docs/develop/api-reference/auth-tokens.md

```diff
@@ -0,0 +1,174 @@
+---
+sidebar_position: 7
+title: Authentication control token API reference information
+sidebar_label: Authentication control tokens
+description: Learn the authentication control token API calls you need to know about and how to use them with Momento services.
+---
+
+import { SdkExampleTabs } from "@site/src/components/SdkExampleTabs";
+// This import is necessary even though it looks like it's un-used; The inject-example-code-snippet
+// plugin will transform instances of SdkExampleTabs to SdkExampleTabsImpl
+import { SdkExampleTabsImpl } from "@site/src/components/SdkExampleTabsImpl";
+
+# Authentication control API reference
+
+<img src="/img/access-tokens.jpg" width="90%" alt="a technical illustration of a bank vault representing security, authorization, and authentication." />
+
+These APIs are used to manage Momento authentication tokens and access. Tokens can be scoped to have one or more roles to give access to one or more caches or topics.
+
+## GenerateAuthToken
+
+Generates a new Momento authentication token with the specified permissions and expiry.
+
+| Name            | Type                       | Description                                                                                       |
+| --------------- |----------------------------|---------------------------------------------------------------------------------------------------|
+| scope           | [TokenScope](#tokenscope)  | The permissions to grant to the new token. Pre-built TokenScope objects are provided by the SDKs. |
+| expiresIn       | Number \| ExpiresIn object | The number of seconds until the token expires or an ExpireIn object calling the `ExpireIn.never()` method.  |
```

Comment:
It can also be a duration defined as minutes or hours by using ExpiresIn.minutes() or ExpiresIn.hours()

Reply:
added